app/main/services/query_builder.py

```diff
@@ -11,6 +13,101 @@
     "supplierName"
 ]
 
+FILTER_FIELDS = [
+    "lot",
+    "serviceTypes",
+    "freeOption",
+    "trialOption",
+    "minimumContractPeriod",
+    "supportForThirdParties",
+    "selfServiceProvisioning",
+    "datacentresEUCode",
+    "dataBackupRecovery",
+    "dataExtractionRemoval",
+    "networksConnected",
+    "apiAccess",
+    "openStandardsSupported",
+    "openSource",
+    "persistentStorage",
+    "guaranteedResources",
+    "elasticCloud"
+]
+
+
+class QueryFilter:
```

Comment:
In Python 2, not inheriting from object creates an 'old style' class which, among other things, doesn't play nicely with type. [new style classes](https://docs.python.org/2/reference/datamodel.html#new-style-and-classic-classes)